cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -598,18 +598,29 @@ pTransaction =
     , subParser "policyid"
         (Opt.info pTransactionPolicyId $ Opt.progDesc "Calculate the PolicyId from the monetary policy script.")
     , subParser "calculate-min-fee"
-        (Opt.info pTransactionCalculateMinFee $ Opt.progDesc "Calculate the minimum fee for a transaction")
-    , subParser "calculate-min-value"
-        (Opt.info pTransactionCalculateMinValue $ Opt.progDesc "Calculate the minimum value for a transaction")
-
+        (Opt.info pTransactionCalculateMinFee $ Opt.progDesc "Calculate the minimum fee for a transaction.")
+    , subParser "calculate-min-required-utxo"
+        (Opt.info pTransactionCalculateMinValue $ Opt.progDesc "Calculate the minimum required UTxO for a transaction output.")
+    , pCalculateMinRequiredUtxoBackwardCompatible
     , subParser "hash-script-data"
-        (Opt.info pTxHashScriptData $ Opt.progDesc "Calculate the hash of script data")
+        (Opt.info pTxHashScriptData $ Opt.progDesc "Calculate the hash of script data.")
     , subParser "txid"
-        (Opt.info pTransactionId $ Opt.progDesc "Print a transaction identifier")
+        (Opt.info pTransactionId $ Opt.progDesc "Print a transaction identifier.")
     , subParser "view" $
-        Opt.info pTransactionView $ Opt.progDesc "Print a transaction"
+        Opt.info pTransactionView $ Opt.progDesc "Print a transaction."
     ]
  where
+  -- Backwards compatible parsers
+  calcMinValueInfo :: ParserInfo TransactionCmd
+  calcMinValueInfo =
+    Opt.info pTransactionCalculateMinValue
+      $ Opt.progDesc "Calculate the minimum value for a transaction"
+
+  pCalculateMinRequiredUtxoBackwardCompatible :: Parser TransactionCmd
+  pCalculateMinRequiredUtxoBackwardCompatible =
+    Opt.subparser
+      $ Opt.command "calculate-min-value" calcMinValueInfo <> Opt.internal
```

Comment:
Are these deprecated?  Should there be a recommendation in the help to use something else instead?

Reply:
Yeah I should include a deprecation warning.